src/data/Site.ts

```diff
@@ -70,15 +70,15 @@ const config: Config = {
 	twitch: new SocialMediaHandle("https://twitch.tv/", "quiltmc"),
 
 	twitter: {
-		handle: new SocialMediaHandle("https://twitter.com/", "quilt_mc"),
+		handle: new SocialMediaHandle("https://x.com/", "quilt_mc"),
 		card: "summary",
 	},
 
 	social: {
 		name: "QuiltMC",
 
 		links: [
-			new URL("https://twitter.com/quilt_mc"),
+			new URL("https://x.com/quilt_mc"),
```

Comment:
```suggestion
			new URL("https://twitter.com/quilt_mc"),
```
x.com still redirects to twitter.com, changing the URL is pointless.